.travis.yml

```diff
@@ -3,20 +3,43 @@ rvm:
   - 2.0.0
   - 2.1
   - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
+  - 2.6
   # optional
   - ruby-head
   - jruby-19mode
   - jruby-head
   - rbx-2
 
+addons:
+  hosts:
+    - ldap.example.org # needed for TLS verification
+
+services:
+  - docker
+
 env:
   - INTEGRATION=openldap
 
 before_install:
   - gem update bundler
 
 install:
-  - if [ "$INTEGRATION" = "openldap" ]; then sudo script/install-openldap; fi
+  - >
+    docker run \
+      --hostname ldap.example.org \
+      --env LDAP_TLS_VERIFY_CLIENT=try \
+      -p 389:389 \
+      -p 636:636 \
+      -v $(pwd)/test/fixtures/seed.ldif:/container/service/slapd/assets/config/bootstrap/ldif/50-bootstrap.ldif \
+      -v $(pwd)/test/fixtures/openldap/retcode.ldif:/container/service/slapd/assets/config/bootstrap/ldif/06-retcodes.ldif \
```

Comment:
My thought was these were different kind of data (the second seems to be configuration of the server return codes). I'll give it a try, but chances are it will replace the files that already exist in the container folder instead of adding them - I'll double check that 👍

Reply:
Indeed files would be replaced when mounting a folder, but the container creators thought about this use case, and provide an empty `/custom` folder where you can add all your ldifs 🎉 

See https://github.com/ruby-ldap/ruby-net-ldap/pull/333/commits/693b210a80302414d79a224455dab79764e9ea66